### PR TITLE
Make HP tuning tutorial more accurate

### PR DESCRIPTION
**Pull request by lostella**
*Description of changes:* Training model for a little longer in the tutorial, reducing the number of trials. I think this makes the notebook a little slower to run, but at the advantage that the displayed results are more meaningful (prediction looks better in the plotted example).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: docs/tutorials/advanced_topics/hp_tuning_with_optuna.md [hunk 1]
```diff
@@ -45,31 +44,22 @@ In general, the datasets provided by GluonTS are objects that consists of three
...
-test_subset = list(islice(dataset.test, 15))
-```
-
-
 This is what the data looks like (first training series, first two weeks of data)
 
+
 ```python
-to_pandas(train_subset[0])[:14 * 24].plot()
+to_pandas(next(iter(dataset.train)))[:14 * 24].plot()
```
Comment (jaheba) [inline docs/tutorials/advanced_topics/hp_tuning_with_optuna.md:57]: There is `first` in `toolz` but it just does the same: https://toolz.readthedocs.io/en/latest/_modules/toolz/itertoolz.html#first

Comment (lostella) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md:57]: I think it’s better to avoid the dependency since it’s just the same

Comment (jaheba) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md:57]: The dependency is already there!

Comment (lostella) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md:57]: is it? https://github.com/awslabs/gluon-ts/blob/dev/requirements/requirements-docs.txt

**Review by jaheba — APPROVED** [review]
Improve is such a wishy washy word.

File: docs/tutorials/advanced_topics/hp_tuning_with_optuna.md (hunk 1 above)
Comment (jaheba) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md:57]: https://github.com/awslabs/gluon-ts/blob/483bc5a8aba6a4a92adc0b8b356fe1f6bccb6e00/requirements/requirements.txt#L7

**Pull request merged**